### DISCUSSION TITLE
maintain folder structure src -> dist?

### DISCUSSION THREAD

**Discussion by stephaniedavidson** [parcel-bundler/parcel#5579] [Q&A]
new parcel user here. I want to compress my files but maintain the folder structure, although not sure parcel is the right tool for this. I see in the docs under "[Multiple entry files](https://parceljs.org/getting_started.html#multiple-entry-files)" you can use something like: `parcel *.html` to grab all html files. When I do that, I get a node error: `×  No entries found.` I think because I'm using my src folder? 

But  this doesn't work either: `parcel src/*.html`, when I run that I get no error but I can't access my subfolder/index.html page. 

My folder structure is like
```
index.html
index.js
/project1
--index.html
--index.css
/project2
--index.html
--index.css
--somejavascript.js
```

What am I missing?

**Comment by mischnic** [comment]
>  can't access my subfolder/index.html page.

You mean `localhost:1234` returns a 404? If there are multiple HTML files, you have to explicitly request one, e.g. `localhost:1234/index.html` because other Parcel doesn't know which one to serve.

**Reply by stephaniedavidson** [reply]
the src/index.html page loads when I go to project1/index.html when running `parcel src/*.html`
